tools/tests/com.liferay.ide.project.core.tests/src/com/liferay/ide/project/core/tests/ImportPluginsSDKProjectTests.java

```diff
@@ -45,6 +35,14 @@
 import org.junit.AfterClass;
 import org.junit.Test;
 
+import com.liferay.ide.core.util.ZipUtil;
```

Comment:
Need to resort the imports, make sure your import-order preferences are correct.